### PR TITLE
transliteration benchmark dataset changes

### PR DESCRIPTION
**Pull request by aswinpradeep**

### CODE REVIEW

File: specs/examples/benchmark-dataset/transliteration-benchmark-dataset/params.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+    "name": "Anuvaad en-hi Transliteration Evaluation Benchmark",
+    "taskType": "transliteration",
+    "description": "Anuvaad Evaluation Benchmark for Low-Resource Transliteration",
+    "languages": {
+        "sourceLanguage": "en",
+        "targetLanguage": "mhil"
```
Comment (aravinth) [inline specs/examples/benchmark-dataset/transliteration-benchmark-dataset/params.json:7]: targetLanguage code seems incorrect.

File: specs/examples/benchmark-dataset/transliteration-benchmark-dataset/data.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+[
+    {
+        "sourceText": "bhaarat-nepaal seema ke kareeb hone ke chalate is pul ka rananeetik mahatv bhee hai",
+        "targetText": "भारत-नेपाल सीमा के करीब होने के चलते इस पुल का रणनीतिक महत्व भी है"
```
Comment (aravinth) [inline specs/examples/benchmark-dataset/transliteration-benchmark-dataset/data.json:4]: Can the records be a sentence or just a single word?

File: specs/examples/benchmark-dataset/transliteration-benchmark-dataset/params.json (hunk 1 above)
Comment (aswinpradeep) [reply specs/examples/benchmark-dataset/transliteration-benchmark-dataset/params.json:7]: fixed

File: specs/examples/benchmark-dataset/transliteration-benchmark-dataset/data.json (hunk 2 above)
Comment (aswinpradeep) [reply specs/examples/benchmark-dataset/transliteration-benchmark-dataset/data.json:4]: ideally its words only, editing now.

**Pull request merged**